util/sample_collections.py

```diff
@@ -139,10 +182,14 @@ def __init__(self, sdb_filename, buffering=BUFFER_SIZE, id_prefix=None):
         self.speech_index = speech_columns[0]
         self.audio_type = self.schema[self.speech_index][MIME_TYPE_KEY]
 
-        transcript_columns = self.find_columns(content=CONTENT_TYPE_TRANSCRIPT, mime_type=MIME_TYPE_TEXT)
-        if not transcript_columns:
-            raise RuntimeError('No transcript data (missing in schema)')
-        self.transcript_index = transcript_columns[0]
+        self.transcript_index = None
+        if labeled is not False:
+            transcript_columns = self.find_columns(content=CONTENT_TYPE_TRANSCRIPT, mime_type=MIME_TYPE_TEXT)
+            if transcript_columns:
+                self.transcript_index = transcript_columns[0]
+            else:
+                if labeled is True:
```

Comment:
`labeled` is always `True` here.

Reply:
Ah, this is a tri-state, isn't it... Maybe lift `Tristate` from `data/lm/generate_package.py` until somewhere in `util/` and use it to avoid these traps?

Reply:
`something is True`, `something is False` and `something is None` are mutually exclusive - which turned out to be quite helpful and easy to grasp. I think I'll keep stuff as it is.